app/controllers/pokemons_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PokemonsController < ApplicationController
+  def index
+    @pokemon = Pokemon.all
+  end
+
+  def show
+    @pokemon = Pokemon.find[params[:id]]
+  end
+
+  def new
+    @pokemon = Pokemon.new
+  end
+
+  def create
+    pokemon = Pokemon.new(pokemon_params)
```

Comment:
store pokemon in an instance variable, so we can use it outside of the method.
`@pokemon = Pokemon.new(pokemon_params)`